libc/src/__support/CPP/type_traits/is_destructible.h

```diff
@@ -21,7 +21,7 @@
 namespace LIBC_NAMESPACE::cpp {
 
 // is_destructible
-#if LIBC_HAS_BUILTIN(__is_destructible)
+#if __has_builtin(__is_destructible)
```

Comment:
We no longer need to include src/__support/macros/config.h here thanks to this change.

Reply:
Done.